cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/Checker.java

```diff
@@ -5,24 +5,25 @@
 
 import java.util.*;
 import java.util.function.Function;
-import java.util.stream.Collectors;
 
 import static com.codahale.metrics.MetricRegistry.name;
 
 /**
  * Created by okutane on 11/08/16.
  */
 public class Checker {
+    static boolean STRICT_MODE = false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "STRICT_MODE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3008)